package/batocera/core/batocera-configgen/configgen/configgen/utils/gamescope.py

```diff
@@ -0,0 +1,103 @@
+from __future__ import annotations
+
+import json
+import logging
+import subprocess
+from contextlib import contextmanager
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from collections.abc import Iterator
+
+    from ..generators.Generator import Generator
+
+_logger = logging.getLogger(__name__)
+
+@contextmanager
```

Comment:
Why are you making this a context manager? You're not using it as one and you never `yield` from it.